server/meshmodel/error.go

```diff
@@ -0,0 +1,14 @@
+package meshmodel
```

Comment:
@humblenginr can we be rid of this package name and use `registry` instead?

Reply:
yes

Reply:
Moved this whole thing into `registration` package